.github/workflows/testing.yml

```diff
@@ -11,11 +11,23 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+
+    # Set up Python 3.10 to match the version of Python used in AWS Lambda
+    - name: Set up Python 10
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.10.12
+
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install -r requirements.txt
-    # - name: Run tests
-    #   run: |
-    #     pytest --pyargs lambda_function --cov lambda_function
+        python -m pip install --upgrade pip git+https://github.com/HERMES-SOC/hermes_core.git moto pytest pytest-cov pytest-astropy
```

Comment:
should these go in requirements.txt?

Reply:
Since they are only needed for development, I created a `requirements.dev.txt` file and changed it to install that one.